tests/unit/handlers/test_web.py

```diff
@@ -0,0 +1,130 @@
+import unittest
+from mindsdb.integrations.libs.api_handler_exceptions import TableAlreadyExists
+from mindsdb.integrations.handlers.web_handler.web_handler import WebHandler
+from mindsdb.integrations.handlers.web_handler.web_handler import CrawlerTable
+from mindsdb.integrations.handlers.web_handler import urlcrawl_helpers as helpers
+from unittest.mock import patch, MagicMock
+import concurrent.futures
+import pytest
+from bs4 import BeautifulSoup
+import pandas as pd
+
+
+class TestWebsHandler(unittest.TestCase):
```

Comment:
Should we add a test for `check_connection()`; I know it probably won't do much here, but it might be good to include it just to ensure it exists and returns the correct response?

Reply:
Yes, good point. Once we find a way for a good healthcheck